app/src/test/java/org/simple/clinic/bloodsugar/history/BloodSugarHistoryScreenUiRendererTest.kt

```diff
@@ -37,4 +37,42 @@ class BloodSugarHistoryScreenUiRendererTest {
     verify(ui).showPatientInformation(patient)
     verifyNoMoreInteractions(ui)
   }
+
+  @Test
+  fun `when blood sugar history is loaded, then show it on the ui`() {
+    // given
+    val bloodSugar = PatientMocker.bloodSugar(
+        uuid = UUID.fromString("bfeed199-ca34-4503-aac9-8903f9501c5b"),
+        patientUuid = patientUuid
+    )
+    val bloodSugars = listOf(bloodSugar)
+
+    // when
+    renderer.render(defaultModel.bloodSugarsLoaded(bloodSugars))
+
+    // then
+    verify(ui).showBloodSugarHistory(bloodSugars)
+    verifyNoMoreInteractions(ui)
+  }
+
+  @Test
+  fun `when both patient and blood sugar history is loaded, then show it on the ui`() {
```

Comment:
This test is not actually necessary since the two are being tested independently.